snapraid-aio-script.sh

```diff
@@ -26,7 +26,7 @@ source "$CONFIG_FILE"
 
 function main(){
   # create tmp file for output
-  true > "$TMP_OUTPUT"
+  exec > "$TMP_OUTPUT"
```

Comment:
Not related to parsing, but is an update to the shellcheck changes. This is more acceptable syntax here.